tarorpc/taro.proto

```diff
@@ -628,3 +635,21 @@ message SendAssetResponse {
 
     int64 total_fee_sats = 5;
 }
+
+message SubscribeSendAssetEventNtfnsRequest {
+}
```

Comment:
Writing down prior comment here re the request of a ID here to permit delivering backlog notifications.

Reply:
Yes, maybe it would be worth adding both the `deliverExisting` boolean and `deliverFrom` timestamps from the `RegisterSubscriber` interface here so we could actually implement it in the porter.

Reply:
I didn't mean to add backlog support in this PR. I figured I would add those fields in a future PR. Would that work for you guys?